.github/workflows/check-unlabeled-issues.yml

```diff
@@ -0,0 +1,60 @@
+name: 'Check unlabeled issues'
+on:
+  schedule:
+    # Run every sunday at midnight
+    - cron: '0 0 * * 0'
+
+jobs:
+  check-unlabeled-issues:
+    runs-on: ubuntu-latest
+
+    env:
+      GH_TOKEN: ${{ secrets.GH_TOKEN }}
```

Comment:
it should use the build-in token. also set workflow permissions explicit

Reply:
> also set workflow permissions explicit

The [GitHub Docs, assigning permissions to jobs](https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs) has more information about this.